src/services/ipfs-service.ts

```diff
@@ -155,4 +166,39 @@ export class IpfsService implements IIpfsService {
 
     return anchorCid
   }
+
+  async importCAR(car: CAR, options: AbortOptions = {}): Promise<void> {
+    const blocks = []
+    for (const block of car.blocks) {
+      blocks.push(block)
+    }
+    await Promise.all(blocks.map(async block => {
+      const cid = block.cid
+      const format = this.codecNames.get(cid.code)
+      const mhtype = this.hasherNames.get(block.cid.multihash.code)
+      await this.ipfs.block.put(block.payload, {
+        format: format,
+        mhtype: mhtype,
+      })
+      await this.ipfs.pin.add(block.cid, {
```

Comment:
ooh I didn't think to pin the blocks in the ceramic-side PR.  Good call, I'll add that to mine